src/neo/SmartContract/StorageItem.cs

```diff
@@ -3,11 +3,12 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Linq;
 using System.Numerics;
 
 namespace Neo.SmartContract
 {
-    public class StorageItem : ISerializable
+    public class StorageItem : ISerializable, IEquatable<StorageItem>
```

Comment:
Why?

Reply:
@shargon

Reply:
I will remove it, it was for detect the problem but the checks was fixed